<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	
	background-color: rgb(237, 242, 254);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
	border-radius: 13px;
	font: 15 10pt &quot;Microsoft YaHei Light&quot;;
	border: 2px solid darkgray;
}
QComboBox::down-arrow{
	back-color: #FFFFF;
}
QComboBox:on{
    padding-top: 3px;
    padding-left: 4px;
}</string>
        </property>
        <item>
         <property name="text">
          <string>Тип пенсии для расчётов</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Страховая</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Накопительная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Государственная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Социальная</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #EE2F53;
	border: 1px ;
	border-radius: 15px;
}</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_old">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
    width: 20 px;
    height: 20 px;
}
QCheckBox{
font: 25 10pt &quot;Microsoft YaHei&quot;;
}
</string>
          </property>
          <property name="text">
           <string>По старости</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_lost">
          <property name="styleSheet">
           <string notr="true">QCheckBox{
	
}
QCheckBox::indicator {
    width: 20 px;
    height: 20 px;
}
QCheckBox{
font: 25 10pt &quot;Microsoft YaHei&quot;;
}</string>
          </property>
          <property name="text">
           <string>Потеря кормильца</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_inv">
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
    width: 20 px;
    height: 20 px;
}
QCheckBox{
font: 25 10pt &quot;Microsoft YaHei&quot;;
}</string>
          </property>
          <property name="text">
           <string>По инвалидности</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>210</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_inval">
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	font: 25 12pt &quot;Microsoft YaHei&quot;;
	font-size: 15px;
	border-radius: 15 px;
	border: 2px gray;
}</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="text">
             <string>Степень инвалидности:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Inval">
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    selection-background-color: darkgray;
}</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_Prem">
          <property name="styleSheet">
           <string notr="true">QLabel{
	font: 25 12pt &quot;Microsoft YaHei&quot;;
	font-size: 15px;
	border-radius: 15 px;
}</string>
          </property>
          <property name="text">
           <string>Кол-во премиальных коэфициентов:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit__prem">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    selection-background-color: darkgray;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_YED">
          <property name="styleSheet">
           <string notr="true">QLabel{
	font: 25 12pt &quot;Microsoft YaHei&quot;;
	font-size: 15px;
	border-radius: 15 px;
}</string>
          </property>
          <property name="text">
           <string>У/eд</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_IPK">
              <property name="styleSheet">
               <string notr="true">QLabel{
	font: 25 12pt &quot;Microsoft YaHei&quot;;
	font-size: 15px;
	border-radius: 15 px;
}</string>
              </property>
              <property name="text">
               <string>Кол-во ИПК:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_IPK">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    selection-background-color: darkgray;
}</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_YED_2">
              <property name="styleSheet">
               <string notr="true">QLabel{
	font: 25 12pt &quot;Microsoft YaHei&quot;;
	font-size: 15px;
	border-radius: 15 px;
}</string>
              </property>
              <property name="text">
               <string>У/eд</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_SIPK">
              <property name="styleSheet">
               <string notr="true">QLabel{
	font: 25 12pt &quot;Microsoft YaHei&quot;;
	font-size: 15px;
	border-radius: 15 px;
}</string>
              </property>
              <property name="text">
               <string>Кол-во СИПК:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_SIPK">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    selection-background-color: darkgray;
}</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_YED_3">
              <property name="styleSheet">
               <string notr="true">QLabel{
	font: 25 12pt &quot;Microsoft YaHei&quot;;
	font-size: 15px;
	border-radius: 15 px;
}</string>
              </property>
              <property name="text">
               <string>У/eд</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #0066B3;
	border: 1px ;
	border-radius: 15px;
	font: 15px Bold Italic;
	font-color: &quot;white&quot;;
	
}</string>
          </property>
          <property name="text">
           <string>Рассчитать размер пенсии:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	border: 2px solid rgb(153, 119, 255);
	border-radius: 15px;
	font: 17px Italic; 
}</string>
      </property>
      <property name="text">
       <string>Результат обработки:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
